couchdb/templates/secrets.yaml

```diff
@@ -18,4 +18,17 @@ data:
 {{- if  .Values.adminHash  }}
   password.ini: {{ tpl (.Files.Get "password.ini") . | b64enc }}
 {{- end -}}
-{{- end -}}
+{{- end }}
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ template "couchdb.fullname" . }}-internal
+  labels:
+    app: {{ template "couchdb.fullname" . }}
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    release: "{{ .Release.Name }}"
+    heritage: "{{ .Release.Service }}"
+type: Opaque
+data:
+  uuid: {{- include "couchdb.uuid" . }}
```

Comment:
I'm not convinced about adding another secret resource here. The secret values `erlangCookie`, `cookieAuthSecret` are similarly internal only. It seems like a good clarification to split internal/external secrets but that could be a separate PR (and likely major version bump).

Reply:
@willholley Thoughts on the right approach here? Would you prefer me to tie this into the existing secret?